hardware/raspberrypi/frequency-management.md

```diff
@@ -8,14 +8,27 @@ For Raspberry Pi 3 Model B+, the PCB technology has been changed to provide bett
 
 The Raspberry Pi 4 Model B continues with the same PCB technology as the Raspberry Pi 3B+ to help dissipate excess heat. There is currently no soft limit defined.
 
-### DVFS on the Raspberry Pi 4B
+### DVFS on the Raspberry Pi 4 (4B, 400 and CM4)
 
-On Raspberry Pi 4 Model B, firmware from late November 2019 onwards implements Dynamic Voltage and Frequency Scaling. This technique (outlined on Wikipedia [here](https://en.wikipedia.org/wiki/Dynamic_voltage_scaling)) allows Raspberry Pi 4B to run at lower temperatures whilst still providing the same performance.
+Raspberry Pi 4 devices implement Dynamic Voltage and Frequency Scaling (DVFS). This technique (outlined on Wikipedia [here](https://en.wikipedia.org/wiki/Dynamic_voltage_scaling)) allows Raspberry Pi 4 devices to run at lower temperatures whilst still providing the same performance.
```

Comment:
I know this was an existing reference to Wikipedia. But going forward post-#1911 I'm going to be stripping these out. Wikipedia is not a valid reference source.

Reply:
It is on many subjects. Certainly that article is fine.

Reply:
Right now, yes. But after someone vandalises the page or someone comes and makes major edits that are just wrong. Potentially not.

Reply:
Wikipedia is certainly "dynamic", but IMHO vandalism wouldn't be likely on a technical topic such as DVFS. I have issues with Wikipedia from time to time, but I do give them credit for their [efforts in mitigating vandalism](https://en.wikipedia.org/wiki/Vandalism_on_Wikipedia), formulating a set of standards and setting out policies that, in the large, yield mostly reliable information. 

But if you'll forgive me, I cannot help but remark on your prioritization of stripping Wikipedia references from the documentation over, say, [updating the schematics](https://github.com/raspberrypi/documentation/issues/1915). Just an observation - not a criticism :)